docs/painless/painless-getting-started.asciidoc

```diff
@@ -119,6 +120,31 @@ GET hockey/_search
 ----------------------------------------------------------------
 // CONSOLE
 
+[float]
+===== Missing values
+
+Currently by default, if a document is missing a numeric field `field`,
```

Comment:
I would describe this a bit differently. I suggest using 2 different paragraphs. The first should describe the current behavior (but it is not necessary to say "currently by default"). This should also not state "returns 0" as it is field type dependent what is returned (and all field types behavior should be described).

The next paragraph can then describe the new option. The last sentence can mention the default behavior will change in a future release.